tests/integration/helpers/cluster.py

```diff
@@ -551,8 +548,41 @@ def __init__(
         self.is_up = False
         self.env = os.environ.copy()
         logging.debug(f"CLUSTER INIT base_config_dir:{self.base_config_dir}")
+        if p.exists(self.instances_dir):
+            shutil.rmtree(self.instances_dir, ignore_errors=True)
+            logging.debug(f"Removed :{self.instances_dir}")
+        os.mkdir(self.instances_dir)
```

Comment:
@qoega, seems like it creates unneeded directories:
```
[tavplubix@dell9510 logs]$ ls test_replicated_database/
_instances_gw0_0  _instances_gw1_0  _instances_gw2_0  _instances_gw3_0  _instances_gw4_0
[tavplubix@dell9510 logs]$ ls test_replicated_database/*
test_replicated_database/_instances_gw0_0:

test_replicated_database/_instances_gw1_0:

test_replicated_database/_instances_gw2_0:

test_replicated_database/_instances_gw3_0:
competing_node  docker.log  dummy_node  keeper1  keeper2  keeper3  main_node  snapshot_recovering_node  snapshotting_node

test_replicated_database/_instances_gw4_0:
```

It's really inconvenient when you have to look for the one that actually contains logs.

Please move this code somewhere, it should not be in the constructor.